src/Settings/SettingsValidator.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/*
+ * This file is part of Flarum.
+ *
+ * For detailed copyright and license information, please view the
+ * LICENSE file that was distributed with this source code.
+ */
+
+namespace Flarum\Settings;
+
+use Flarum\Foundation\AbstractValidator;
+
+class SettingsValidator extends AbstractValidator
+{
+    /**
+     * Make a new validator instance for this model.
+     *
+     * @param array $attributes
+     * @return \Illuminate\Validation\Validator
+     */
+    protected function makeValidator(array $attributes)
+    {
+        // These rules apply to all settings.
```

Comment:
```suggestion
        // Entries in the default DB settings table are limited to 65,000
        // characters. We validate against this to avoid confusing errors.
```